docs/connectors/rhoas-cli-getting-started-connectors/README.adoc

```diff
@@ -102,17 +102,21 @@ For the example in this guide, you create a source connector that sends data fro
 
 Use this guide to complete the following tasks:
 
-* {base-url}{getting-started-rhoas-cli-url-connectors}#proc-create-connector-namespace_connectors-rhoas-cli[Create a namespace to host your {product-connectors} instances]
-* {base-url}{getting-started-rhoas-cli-url-connectors}#proc-building-connector-configuration-cli_connectors-rhoas-cli[Build a configuration file for each type of connector that you want to create]
-* {base-url}{getting-started-rhoas-cli-url-connectors}#proc-create-connector-instances_connectors-rhoas-cli[Create {connectors} instances]
+* {base-url}{getting-started-rhoas-cli-url-connectors}#_75aa250f-388e-4a3c-86b6-d4dc59741df1[Create a namespace to host your {product-connectors} instances]
+
+* {base-url}{getting-started-rhoas-cli-url-connectors}#_c0160a2f-7620-4e48-8bcd-887923820323[Build a configuration file for each type of connector that you want to create]
+
+* {base-url}{getting-started-rhoas-cli-url-connectors}#_3c861147-85d5-49de-8d5e-3213f7a868b9[Create {connectors} instances]
+
+* {base-url}{getting-started-rhoas-cli-url-connectors}#_ecb48393-1ae9-4eb8-8583-1eaacd46ec0f[Managing {connectors} instances]
 
 .Prerequisites
 
-* You have a {org-name} account.
-* You've installed the latest version of the `rhoas` CLI. See {base-url}{installation-guide-url-cli}[Installing and configuring the rhoas CLI^].
+* You have an {org-name} account.
+* You've installed the latest version of the `rhoas` CLI as described in {base-url}{installation-guide-url-cli}[Installing and configuring the OpenShift Application Services CLI^].
```

Comment:
`{org-name}` resolves to `Red Hat` downstream, so should say as `a {org-name}`. 

Suggest adding a downstream-only `ifndef::community[]`condition to avoid `a Application Services` upstream like in other guides.